cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -89,6 +89,10 @@ if (DEFINED ENV{GRPC_HOME})
   set(GRPC_HOME "$ENV{GRPC_HOME}")
 endif()
 
+if (DEFINED ENV{CARES_HOME})
```

Comment:
It would be nice to keep those directives in lexographical order.

Reply:
Agreed, fixed :)